libraries/plugin.rb

```diff
@@ -44,9 +44,6 @@ class Resource::JenkinsPlugin < Resource::LWRPBase
               default: :latest
     attribute :source,
               kind_of: String
-    attribute :install_deps,
```

Comment:
This should remain and instead you should throw a log warning that the attribute is deprecated and will be removed later. That way people can upgrade without breaking the world.

Reply:
This should be fixed now.